.github/workflows/make.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     - name: Initialize submodules
       run: |
         git submodule update --init --recursive
+        git fetch --prune --unshallow
```

Comment:
This is likely better to be handled by setting `fetch-depth: 0` in the `checkout` step. By default the `checkout` action clones with `fetch-depth: 1` https://github.com/actions/checkout?tab=readme-ov-file#usage